test/setActions.test.ts

```diff
@@ -37,14 +37,22 @@ describe("Set the Actions of the Circuit", () => {
     175177,
   );
 
+  const provider = new ethers.providers.JsonRpcProvider(
+    "https://yellowstone-rpc.litprotocol.com/",
```

Comment:
Not really that important here. But better to use our constant for this and the network keys to avoid typos and get some validation when these things change in the underlying SDK (for example, when we update to a new version that drops datil)